x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/action_clone.test.ts

```diff
@@ -0,0 +1,254 @@
+        },
+        analysis: {
+          classification: {
+            dependent_variable: 'y',
+            num_top_classes: 2,
+            prediction_field_name: 'y_prediction',
+            training_percent: 2,
+            randomize_seed: 6233212276062807000,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '350mb',
+        allow_lazy_start: false,
+      };
+
+      expect(isAdvancedConfig(advancedClassificationJob)).toBe(true);
+    });
+
+    test('should detect advanced outlier_detection job', () => {
+      const advancedOutlierDetectionJob = {
+        description: "Outlier detection job with 'glass' dataset",
+        source: {
+          index: ['glass_withoutdupl_norm'],
+          query: {
+            // TODO check default for `match`
+            match_all: {},
+          },
+        },
+        dest: {
+          index: 'dest_glass_1',
+          results_field: 'ml',
+        },
+        analysis: {
+          outlier_detection: {
+            compute_feature_influence: false,
+            outlier_fraction: 0.05,
+            standardization_enabled: true,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: ['id', 'outlier'],
+        },
+        model_memory_limit: '1mb',
+        allow_lazy_start: false,
+      };
+      expect(isAdvancedConfig(advancedOutlierDetectionJob)).toBe(true);
+    });
+
+    test('should detect a custom query', () => {
+      const advancedRegressionJob = {
+        description: "Regression job with 'electrical-grid-stability' dataset",
+        source: {
+          index: ['electrical-grid-stability'],
+          query: {
+            match: {
+              custom_field: 'custom_match',
+            },
+          },
+        },
+        dest: {
+          index: 'dest_grid_1',
+          results_field: 'ml',
+        },
+        analysis: {
+          regression: {
+            dependent_variable: 'stab',
+            prediction_field_name: 'stab_prediction',
+            training_percent: 20,
+            randomize_seed: -2228827740028660200,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '150mb',
+        allow_lazy_start: false,
+      };
+
+      expect(isAdvancedConfig(advancedRegressionJob)).toBe(true);
+    });
+
+    test('should detect custom analysis settings', () => {
+      const config = {
+        description: "Classification clone with 'bank-marketing' dataset",
+        source: {
+          index: 'bank-marketing',
+        },
+        dest: {
+          index: 'bank_classification4',
+        },
+        analyzed_fields: {
+          excludes: [],
+        },
+        analysis: {
+          classification: {
+            dependent_variable: 'y',
+            training_percent: 71,
+            maximum_number_trees: 1500,
```

Comment:
Looks like this is now `max_trees`

Reply:
fixed in https://github.com/elastic/kibana/pull/59791/commits/44cc41ba0909eac4595378a0769474c474b1f188